odxtools/envdatadesc.py

```diff
@@ -19,15 +19,13 @@ class EnvironmentDataDescription(DopBase):
 
     def __init__(self,
                  *,
-                 env_datas: List[EnvironmentData],
                  env_data_refs: List[OdxLinkRef],
-                 param_snref: Optional[str] = None,
-                 param_snpathref: Optional[str] = None,
+                 param_snref: Optional[str],
+                 param_snpathref: Optional[str],
                  **kwargs):
         super().__init__(**kwargs)
 
         self.bit_length = None
-        self.env_datas = env_datas
```

Comment:
What happened here?

Reply:
the XSD allows only to specify ENV-DATA-REFS, so we should not take a list of full  environment data objects here. I can factor this out, if wanted...